schema/Review.schema.yaml

```diff
@@ -0,0 +1,30 @@
+title: Review
+'@id': schema:Review
+extends: CreativeWork
+role: primary
+status: stable
+category: metadata
+description: A review of an item, e.g of an Article, or SoftwareSourceCode.
+$comment: |
+  Use the `content` property for the structured content of the review in
+  preference to the schema.org `text` or `reviewBody` properties which
+  have plain text as the expected type of this property.
+properties:
+  itemReviewed:
+    '@id': schema:itemReviewed
+    description: The item that is being reviewed.
+    $ref: Thing
+  reviewAspect:
```

Comment:
Would this work for references to lines of code as well? E.g., what I'm thinking is that for code review, we often place them in specific spots (akin to this review). Another example is hypothes.is annotation that keeps track of the location on the page. If we just have a Review with an "aspect" that is unlinked from specific comments with different locations, we lose this location.

So I'm wondering if a comment in fact needs some other kind of metadata to indicate a location or context.

Reply:
Yes, that is a good point. How about adding a `commentAspect: Text` property to `Comment`. That could take the form of `some_code.py#L10-L20`, `{"line": 10, "column": 4}`, `whatever-hypothesis-use-to-mark location-of-comment-in a page` etc

Reply:
That would do the trick!